static/app/views/replays/detail/sideTabs.tsx

```diff
@@ -0,0 +1,29 @@
+import NavTabs from 'sentry/components/navTabs';
+import {t} from 'sentry/locale';
+import useUrlParams from 'sentry/utils/replays/hooks/useUrlParams';
+
+type Props = {};
+
+const TABS = {
+  video: t('Replay'),
+  tags: t('Tags'),
+};
+
+function SideTabs({}: Props) {
```

Comment:
This is kind of the replacement for `asideTabs.tsx and `asideTabs_v2.tsx`.

Both of those were trying to handle the tabs as well as the tab-content, with all the wrapping styles and stuff. 

So now we've got the tabs here, and `layout/index.tsx` picks the content to show.